### PR TITLE
WIP Hawkular UI

### PR DESCRIPTION
**Pull request by jotak**
First iteration for a new UI. The initial purpose is to provide a simple and limited interface mostly for debugging purpose / "see that it works". The promoted UI is still Grafana.

It's pushed to a specific branch to eventually allow more cooperation & testing

Currently it displays 1 single chart with a metric chosen from a list + the good old Status page.

Still to do:
- [ ] MANDATORY: Get rid of babel in browser (supposes scripted build/packaging?)
- [ ] MANDATORY: Don't load remote JS/CSS
- [ ] Unit tests
- [ ] Use existing chart lib by @mtho11 (= so, switch to angular 1.x )
- [ ] Manage non numeric series (note there's avail. charts in https://github.com/hawkular/hawkular-ui-components )
- [ ] Display notice about the purpose of this & link to Grafana installation
- [ ] Show config flags?
- [ ] Manage authentication?
- [ ] Grafana export?
- [ ] Add a form to push datapoints? (remember: it's for debug)

### CODE REVIEW

**Comment by jotak** [comment]
Note: it's WIP but can be merged as it's in a dedicated branch

**Comment by burmanm** [comment]
Why couldn't we target master instead?

**Comment by jotak** [comment]
@burmanm I would eventually appreciate collaboration of some JS / UX expert people :)

**Comment by abonas** [comment]
adding screenshot(s) to UI related PRs is very helpful

**Comment by burmanm** [comment]
@jotak Feedback is faster when we have it on the "shipping product" instead of some buried branch ;)

**Comment by jotak** [comment]
Screenshots: http://imgur.com/DuArpQ4.png?1
http://i.imgur.com/XPLJtqb.png?1

File: api/metrics-api-jaxrs/src/main/webapp/static/css/main.css [hunk 1]
```diff
@@ -1,3 +1,15 @@
+.navbar-fixed-top {
```
Comment (gbaufake) [inline api/metrics-api-jaxrs/src/main/webapp/static/css/main.css]: Travis is checking this file on PR and it is not finding the license on it

Comment (jotak) [inline api/metrics-api-jaxrs/src/main/webapp/static/css/main.css]: @gbaufake yes I've seen that. Will update it later (currently replacing react to angular, so anyway I'll have to regenerate all licenses)

**Comment by stefannegrea** [comment]
@jotak, amazing work! 

Could we place this code on a separate module? The JAX-RS project is in the process of a total transformation, at the end of next month most of the code will be touched/moved/replaced/shuffled/deleted.

**Comment by jotak** [comment]
@stefannegrea , if there's a new vertx-web or whatever http-server module I suppose this static webapp code will have to be moved there, and vertx router will have to be adapted to serve it, so why not waiting that it's in place before moving it elsewhere?

**Comment by jotak** [comment]
Closing, replaced with angular 2 (PR coming soon)

**Pull request closed without merging**